### PR TITLE
import missing changes

### PR DESCRIPTION
**Pull request by cassus**

### CODE REVIEW

**Comment by cassus** [comment]
@peterlangmar have a look pls!

**Comment by peterlangmar** [comment]
ez mi?

File: Culture and Learning.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Lab / Culture and Learning
+Purpose: Unspecified.
+
+## Roles
+
+### Lead Link - Tojas
```
Comment (peterlangmar) [inline Culture and Learning.md]: @EggDice tök jó lenne, ha nyomnád, mert a valóságban is te nyomod

File: Sales.md [hunk 2]
```diff
@@ -7,3 +7,14 @@ Purpose: Unspecified.
...
 ### Facilitator -
 ### Secretary -
+
+### Sales - Petak
+Purpose: Unspecified.
+
+### Biz analyst -
+Purpose: Unspecified.
+
+### Estimator -
```
Comment (peterlangmar) [inline Sales.md]: Anybody could be in this role. One person could represent the role, moreover as some of you spend minimal time in this role, it is not a must to attend to the governance meetings.

**Pull request merged**